cogs/socials.py

```diff
@@ -255,15 +255,25 @@ async def gay(self, ctx, user=None, border=False, server_avatar=True):
     @slash_command()
     @option("text", str, description="What do you want to tell Paw?")
     # @commands.cooldown(1, 30, commands.BucketType.user)
-    async def gpt(self, ctx: discord.ApplicationContext, text):
+    async def gpt(self, ctx: discord.ApplicationContext, text: str):
         """ Talk to Paw! """
-        messages = await ctx.channel.history(limit=50).flatten()
         await ctx.defer()
-        messages.reverse()
+        messages = await ctx.channel.history(limit=50, oldest_first=True).flatten()
```

Comment:
Just remove the oldest_first from this and keep the messages.reverse(), if you print each message as it gets appended you can see that it has all kinds of orders, but never a chronological one

Reply:
Oh